templates/easy_admin/Participation/conference.html.twig

```diff
@@ -0,0 +1,8 @@
+<a href="{{ path('easyadmin', {
+        'entity': item.conference.guessConferenceType,
```

Comment:
I think you can retrieve the current entity class in the `easyadmin` object, and so the `guessConferenceType` could be removed

Reply:
I don't think so. How easy admin could guess the **virtual** entity here?

Reply:
Indeed, I was pretty that it can retrieve it, but in fact, there are virtual entities. My bad

Reply:
Unfortunately if I try to access the entity with `link_parameters.entity` it will always be "PassedConference", even if it should be a "NextConference" or "ExcludedConference".

But maybe there is an other way I didn't think about ?